### DISCUSSION TITLE
symfony lancer des appels téléphoniques en API

### DISCUSSION THREAD

**Discussion by Yousra2023YY** [symfony/symfony#51512] [Q&A]
Sur un projet  symfony version 3.

Comment ou quelles les méthodes utilisées  pour effectuer des opérations « RESTFULL » pour le lancement des appels téléphonique !??? lancer un appel téléphonique POST à partir d une url

In a project symfony version three 3. What can I use to implement an API post with an URL to raise some phone calls when I click on a PHONE number on table list CRM?

Please give me a code example to display how I implement...

SVP give me des code exemples de cette impléméntation!....

**Comment by alexandre-daubois** [comment]
If Symfony can be upgraded to version 4.4, then [the HttpClient component](https://symfony.com/doc/current/http_client.html) is the solution for making HTTP calls. Otherwise, there are recognized alternatives [such as GuzzleHttp](https://packagist.org/packages/guzzlehttp/guzzle). Otherwise, as a "last resort", there's always [cURL](https://www.php.net/manual/fr/book.curl.php).
